distributed/deploy/runner.py

```diff
@@ -0,0 +1,227 @@
+import asyncio
+import sys
+import threading
+from contextlib import suppress
+from enum import Enum
+from typing import Dict
+
+from ..core import CommClosedError, Status, rpc
+from ..scheduler import Scheduler
+from ..utils import LoopRunner, import_term, sync, thread_state
+from ..worker import Worker
+
+
+class Role(Enum):
```

Comment:
I think I would prefer a term like "type". In deploy environments, role reminds me of some IAM entity. Maybe that's just me :)

Reply:
Perhaps `ServerRole` or `ServerType` would be better here.

Reply:
Both would work for me. Dealer's choice